front-end/src/components/Post/Poll/index.tsx

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import React from 'react';
+
+import { usePostVotesQuery } from '../../../generated/graphql';
+import Card from '../../../ui-components/Card';
+import FilteredError from '../../../ui-components/FilteredError';
+import CouncilSignals from './CouncilSignals';
+import OffChainSignals from './OffChainSignals';
+import { AYE, NAY } from './votes';
+
+interface Props {
+	postId: number
+}
+
+const Poll = ({ postId }: Props) => {
+	const { data, error, refetch } = usePostVotesQuery({ variables: { postId } });
+	let ayes = 0;
+	let nays = 0;
+
+	data?.post_votes?.forEach(vote => {
```

Comment:
you can use `({vote}) => ` so that the `vote` is directly accessible, no need to double `vote.vote` below.

Reply:
In this loop, I'd add a check for the user's address, and pass the `ownVote` as prop.
Also we'll need a refetch.

Reply:
done